src/server/Document/component.jsx

```diff
@@ -73,6 +74,15 @@ const Document = ({
           />
         )}
         {scriptsAllowed && scriptTags}
+        {scriptsAllowed && (
+          <script
+            type="text/javascript"
+            /* eslint-disable-next-line react/no-danger */
```

Comment:
As discussed the other day with @tochwill I think it would be good a good convention to explain any uses of dSIH, e.g. (feel free to amend)
```suggestion
            // Justification:
            // - we need this to be a blocking script that runs before the page first renders
            // - the content is static text so there is no real XSS risk
            /* eslint-disable-next-line react/no-danger */
```